### DISCUSSION TITLE
Disable playlist playback

### DISCUSSION THREAD

**Discussion by malekhin** [iina/iina#3935] [Q&A] [answered]
If there are multiple videos in a folder, IINA plays them in order as a playlist. Is it possible to disable automatic play of all files in a folder as a playlist?

**Comment by low-batt** [comment, marked as answer]
Try this:
- Open IINA
- Click on `Preferences...` under the `IINA` menu
- Scroll to the `Playlist` section
- Uncheck `Add files in the same folder automatically`
- **Restart** IINA to active the new setting

![discussion-3935](https://user-images.githubusercontent.com/86170219/188323662-5493a61a-6866-4e04-959a-47835c0ccac3.jpg)

There is also `Play next item automatically` if you wanted the list populated, but did not the next title to automatically start playing.
